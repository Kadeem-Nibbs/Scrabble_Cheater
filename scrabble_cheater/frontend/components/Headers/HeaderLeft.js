import React, { Component } from 'react'
import { Header } from 'semantic-ui-react'

class HeaderLeft extends Component {
  render() {  
    return(
      <Header as='h3'>
        Get answers for Words with Friends or Scrabble!
        <Header.Subheader>
           <h4>Click on a tile below, select a direction, and enter a word.</h4>
        </Header.Subheader>
      </Header>
    )
  } 
}

export default HeaderLeft