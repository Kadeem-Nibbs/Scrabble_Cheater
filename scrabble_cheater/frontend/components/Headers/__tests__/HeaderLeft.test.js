import React from 'react'
import HeaderLeft from '../HeaderLeft'

test('render HeaderLeft', () => {
    const wrapper = shallow(<HeaderLeft />)
    expect(wrapper).toMatchSnapshot()    
});
