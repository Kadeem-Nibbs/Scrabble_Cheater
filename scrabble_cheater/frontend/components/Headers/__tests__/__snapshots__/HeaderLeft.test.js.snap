// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render HeaderLeft 1`] = `
<Header
  as="h3"
>
  Get answers for Words with Friends or Scrabble!
  <HeaderSubheader>
    <h4>
      Click on a tile below, select a direction, and enter a word.
    </h4>
  </HeaderSubheader>
</Header>
`;
